scripts/python/migrate_data_2024.py

```diff
@@ -0,0 +1,2683 @@
+                                         database=db,
+                                         user=user,
+                                         port=port,
+                                         password=password)
+
+    try:
+        if connection.is_connected():
+            cursor = connection.cursor()
+            cursor.execute(sql_query_user)
+            data = cursor.fetchall()
+            if len(data) != 0:
+                for row in data:
+                    result[row[0]] = { 'stable_id':row[1],
+                                       'name':row[2],
+                                       'description':row[3],
+                                       'source':row[4] }
+
+    except Error as e:
+        print("Error while connecting to MySQL", e)
+    finally:
+        if connection.is_connected():
+            cursor.close()
+            connection.close()
+
+    return result
+
+"""
+    Return organ data
+    This data is going to be stored as historical data
+"""
+def dump_organ(host, port, db, user, password):
+    result = {}
+
+    sql_query_user = f""" SELECT organ_id, name
+                          FROM organ """
+    
+    connection = mysql.connector.connect(host=host,
+                                         database=db,
+                                         user=user,
+                                         port=port,
+                                         password=password)
+
+    try:
+        if connection.is_connected():
+            cursor = connection.cursor()
+            cursor.execute(sql_query_user)
+            data = cursor.fetchall()
+            if len(data) != 0:
+                for row in data:
+                    result[row[0]] = row[1]
+
+    except Error as e:
+        print("Error while connecting to MySQL", e)
+    finally:
+        if connection.is_connected():
+            cursor.close()
+            connection.close()
+
+    return result
+
+def dump_ontology(host, port, db, user, password, attribs):
+    result_disease = {}
+
+    sql_query_disease = """ SELECT d.disease_id, d.ontology_term_id, d.mapped_by_attrib, o.ontology_accession, o.description
+                            FROM disease_ontology_mapping d
+                            LEFT JOIN ontology_term o ON d.ontology_term_id = o.ontology_term_id
+                        """
+
+    connection = mysql.connector.connect(host=host,
+                                         database=db,
+                                         user=user,
+                                         port=port,
+                                         password=password)
+
+    try:
+        if connection.is_connected():
+            cursor = connection.cursor()
+            cursor.execute(sql_query_disease)
+            data_disease = cursor.fetchall()
+            for row in data_disease:
+                # Fetch ontology (MONDO) info
+                url = f"https://www.ebi.ac.uk/ols4/api/search?q={row[3]}&ontology=mondo&exact=1"
```

Comment:
Suggesting that this url is in the get_mondo function like you have for get_omim also